core/src/test/scala/com/rackspace/com/papi/components/checker/ValidatorWADLRelevantRolesSuite.scala

```diff
@@ -0,0 +1,116 @@
+/** *
+  * Copyright 2017 Rackspace US, Inc.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+package com.rackspace.com.papi.components.checker
+
+import com.rackspace.cloud.api.wadl.Converters._
+import com.rackspace.com.papi.components.checker.RunAssertionsHandler.ASSERT_FUNCTION
+import com.rackspace.com.papi.components.checker.servlet.{CheckerServletRequest, CheckerServletResponse}
+import com.rackspace.com.papi.components.checker.step.results.Result
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+
+import scala.collection.JavaConversions._
+import scala.xml.Elem
+
+@RunWith(classOf[JUnitRunner])
+class ValidatorWADLRelevantRolesSuite extends BaseValidatorSuite {
```

Comment:
This is a pretty light test suite since I only tested the behavior of the few lines of XSL code I added. I don't think it's necessary to stress the state machine since its construction was not touched for this work. Having said that, I know that I'm not familiar enough with `api-checker` to say that these tests cover all of the effects of the code change, so I fully expect someone to call me out and tell me to test certain things that I missed.